core-feature-pack/common/src/main/resources/modules/system/layers/base/org/bouncycastle/bcprov/main/module.xml

```diff
@@ -34,5 +34,9 @@
             <with-class name="org.bouncycastle.pqc.jcajce.provider.BouncyCastlePQCProvider"/>
         </service>
     </provides>
+    <dependencies>
+        <module name="java.sql"/>
+        <module name="java.naming"/>
+    </dependencies>
```

Comment:
This is completely a nitpick and should not prevent this to be merged, but better if we keep the dependencies alphabetically when possible